.eleventy.js

```diff
@@ -0,0 +1,19 @@
+module.exports = function (eleventyConfig) {
+  eleventyConfig.addPassthroughCopy("assets");
+  let pathPrefix = "/";
+
+  if (process.env.BASEURL) {
+    pathPrefix = process.env.BASEURL;
+  }
+
+  return {
+    pathPrefix: pathPrefix,
+    templateFormats: ["md", "njk", "html", "liquid"],
+    markdownTemplateEngine: "liquid",
+    htmlTemplateEngine: "liquid",
+    dir: {
+      layouts: "_layouts",
+      data: "_data",
+    },
```

Comment:
The layouts and data worked without these settings, but making them explicit for clarity.